blackbox/docs/general/ddl/alter-table.rst

```diff
@@ -32,6 +32,35 @@ In order to set a parameter to its default value use ``reset``::
     cr> alter table my_table reset (number_of_replicas);
     ALTER OK, -1 rows affected (... sec)
 
+.. _alter_change_number_of_shard:
+
+Changing Number of Shards
+-------------------------
+
+It is possible to change the number of primary shards of a table
+
+..    todo: remove comments once feature is implemented
+..    cr> alter table my_table set (number_of_shards = 2);
+..    ALTER OK, -1 rows affected (... sec)
+
+.. Note::
+    Currently we only support table shrinking, meaning that
+    the requested number of shards must be less than the
+    current number of the primary shards. Moreover, the requested number
+    of primary shards must be a factor of the current number of primary shards.
+    For example, a table with 8 primary shards can be shrunk
+    into 4, 2 or 1 primary shards.
+    If the number of shards of a table is a prime number,
```

Comment:
I think this should a warning on its own as it has bigger implications than a "note" would imply

Reply:
done